storage/asynchbase/src/main/java/net/opentsdb/storage/Tsdb1xMultiGet.java

```diff
@@ -448,13 +469,13 @@ void onError(final Throwable t) {
       node.pipelineContext().queryContext().logError(node, 
           "Multiget query failed with exception: " + t.getMessage());
       state = State.EXCEPTION;
-        if (!node.push()) {
+      if (!node.push()) {
         current_result.setException(t);
         final QueryResult result = current_result;
         current_result = null;
         outstanding.set(0);
         node.onNext(result);
-      } else {
+      } else if (!node.pipelineContext().queryContext().isClosed()) {
```

Comment:
call error if query context is closed?

Reply:
Nope, cause we no longer have any place to send the error.